core/primitives/src/ed25519.rs

```diff
@@ -458,7 +458,7 @@ impl TraitPair for Pair {
 			Err(_) => return false
 		};
 
-		match public_key.verify(message.as_ref(), &sig) {
+		match public_key.verify::<sha2::Sha512>(message.as_ref(), &sig) {
```

Comment:
Can you elaborate on those? Are these changes equivalent?

Reply:
It should be.

Reply:
@tomaka, can you please confirm this?

Reply:
I tried to check it myself and it seems that it should be fine. However, I am not 100%.